library/init/core.lean

```diff
@@ -142,6 +142,7 @@ inductive false : Prop
 
 inductive empty : Type
 
+/-- `not P`, with notation `¬ P`, is the Proposition which is true if and only if `P` is false. It is internally represented as `P → false`. -/
```

Comment:
```suggestion
/-- `not P`, with notation `¬ P`, is the `Prop` which is true if and only if `P` is false. It is internally represented as `P → false`. -/
```
to be consistent with below (or change that one)

Reply:
I've incorporated this into my suggestion below.